Creating new applications/rails.md

```diff
@@ -0,0 +1,12 @@
+# Checklist for creating a new DACS rails application
+
+- [ ] The application has a github repo in the pulibrary organization.
+- [ ] The application uses `main` as the default branch
+- [ ] The application does not use Minitest (e.g. when creating your application, you use the `-T` option for `rails new`)
```

Comment:
can we move this step before the rspec installation step?

Reply:
can we move minitest part before 10. rspec installation?

Reply:
I actually fixed it this time 😅, apologies for misunderstanding. 🐬